src/modules/home/home.tsx

```diff
@@ -1,191 +1,475 @@
-import './home.scss';
-import { useCallback } from 'react';
-import { useHistory } from 'react-router-dom';
+import "./home.scss";
+import { useCallback } from "react";
+import { useHistory } from "react-router-dom";
+
+function handleAccordionClick(headerId, contentId){
+  var accordionTopButton = document.getElementById(headerId);
```

Comment:
Use let keyword. Var variables belong to the global scope.

Reply:
Done, here and in confirmation.tsx